.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: required
+services:
+  - docker
+
+env:
+  COMPOSE_VERSION: 1.9.0
+
+language: node_js
+node_js:
+  - "6"
+  - "6.1"
```

Comment:
this will create a matrix of tests (and run tests for each entry of the matrix). So only 6 is useful actually (may be 4 if we want to support it but there's no reasons for now)